### PR TITLE
Injector failed to inject daprd due to crash.

### PR DESCRIPTION
**Pull request by ls-2018**
# Description

- https://github.com/dapr/dapr/issues/6238 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by ls-2018** [comment]
test

**Review by ItalyPaleAle — CHANGES_REQUESTED** [review]
Thanks for suggesting this PR :)

Sadly, we cannot make this change. This is something we considered one year ago already, but it is not an option for us. You can read more here: https://github.com/dapr/dapr/issues/4171

TLDR: Setting this to `Fail` may make it impossible to restart K8s if the entire cluster goes down.

As per your issue, I have responded here with some advice on how you can address your issue: https://github.com/dapr/dapr/issues/6238#issuecomment-1507868995

**Comment by ls-2018** [comment]
Thank you for your answer,  
I turned on this switch, but it didn't seem to take effect, and the operator reported an error. I don't know what caused it for now.And these Pods have been running for a while.
image
image

image
